index.bs

```diff
@@ -501,31 +498,24 @@ Abstract Operations {#abstract-operations}
 
 Automation {#automation}
 ==========
-This section extends the [=automation=] section defined in the Generic Sensor API [[GENERIC-SENSOR]]
-to provide mocking information about the device's physical orientation in relation to a three
-dimensional Cartesian coordinate system for the purposes of testing a user agent's implementation
-of {{AbsoluteOrientationSensor}} and {{RelativeOrientationSensor}} APIs.
 
-<h3 id="mock-orientation-sensor-type">Mock Sensor Type</h3>
+This section extends [[GENERIC-SENSOR#automation]] by providing [=Orientation Sensor=]-specific virtual sensor metadata.
 
-The {{AbsoluteOrientationSensor}} class has an associated [=mock sensor type=] which is
-<a for="MockSensorType" enum-value>"absolute-orientation"</a>, its [=mock sensor reading values=]
-dictionary is defined as follows:
+<h3 dfn> Absolute Orientation Sensor automation</h3>
 
-<pre class="idl">
-  dictionary AbsoluteOrientationReadingValues {
-    required FrozenArray&lt;double>? quaternion;
-  };
-</pre>
+The [=per-type virtual sensor metadata=] [=map=] must have the following [=map/entry=]:
+: [=map/key=]
+:: "`absolute-orientation`"
+: [=map/value=]
+:: A [=virtual sensor metadata=] whose [=virtual sensor metadata/virtual sensor type=] is [[#absoluteorientationsensor-model|AbsoluteOrientationSensor]] and [=reading parsing algorithm=] is [=parse quaternion reading=].
```

Comment:
You shouldn't need the `[[#section|term]]` syntax for something defined in the same spec. I believe you had to do this because Bikeshed complained if you tried to use `[=AbsoluteOrientationSensor=]`, which makes sense since you're referring to an IDL definition. See https://speced.github.io/bikeshed/#autolink-microsyntax as well as all other uses of `AbsoluteOrientationSensor` in the spec.

With that said, a virtual sensor type needs to be a [sensor type](https://w3c.github.io/sensors/#model-sensor-type), which is different from an IDL interface. See how https://w3c.github.io/gyroscope/#automation refers to the [Gyroscope sensor type](https://w3c.github.io/gyroscope/#gyroscope-sensor-type), not the [`Gyroscope` IDL interface](https://w3c.github.io/gyroscope/#gyroscope).

Curiously, this spec doesn't actually define sensor types for absolute/relative orientation sensor. I'm fixing this in #80 -- you need to either rebase this PR on top of it or wait for it to be merged.

Reply:
Thanks for the change. Now (https://github.com/w3c/orientation-sensor/pull/78/commits/8408a21c8d9be88c37a55d3bf8c729258e30a094) sensor names point to sensor type.